docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -1489,26 +1498,22 @@ invalid:
   - daml-prim
   - daml-stdlib
   - daml3-script
-  - ../../dependencies/depA-1.0.0.dar
-  - ../../dependencies/depB-1.0.0.dar
+  data-dependencies:
+  - dependencies/dep-1.0.0.dar # Cannot downgrade dep-2.0.0 to dep-1.0.0
   ...
 
-At the moment, ``daml build`` does not check for valid dependencies - checks
-are instead performed at upload time to a participant.
+If you try to build this package, the typechecker will error on a package ID
+mismatch for the Dep:AdditionalData field, because the Dep:AdditionalData
+reference in this case has changed to a package that is not a legitimate upgrade
+of the original.
```

Comment:
```suggestion
If you try to build this package, the typechecker returns an error on a package ID
mismatch for the Dep:AdditionalData field, because the Dep:AdditionalData
reference in this case has changed to a package that is not a legitimate upgrade
of the original.
```